### PR TITLE
EWL-3923 Tag releases

### PR DESCRIPTION
**Pull request by froboy**
## Ticket(s)

NA

**Github Issue**

N/A - Requested by Manning.

**Jira Ticket**

- [EWL-3923: Deal with gulp deploy](https://issues.ama-assn.org/browse/EWL-3923)

## Description

As per [this comment](https://issues.ama-assn.org/browse/EWL-3923?focusedCommentId=93364&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-93364), we're now setting up the following structure:

- `develop` - the (uncompiled) working branch (exists currently)
- `gh-pages` - the compiled, publicly accessible “production” artifact (exists currently, deployable with `gulp deploy`)
- `dev-assets` - the compiled, drupal-consmed artifact for testing (exists currently, deployable with `gulp drupal-deploy`)
- `master` - the compiled production release that gets tagged. (unmaintained up to now, new process with `gulp release` will take care of this)

## To Test

- Review README
- Try pushing a new release if you feel comfortable.

## Relevant Screenshots/GIFs

NA

## Remaining Tasks

Remaining tasks?

## Additional Notes

Anything more to add?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,21 +47,15 @@ _this should only be run on the `develop` branch`*`_
 
 - Make sure you are running the most up-to-date code
   - Updates will be rejected if they are non-fast-forward
-- `gulp release` to build the files correctly, update version information, cut a tag, and deploy the files to `gh-pages`
+- Update `package.json` with the new version number.
+- `gulp release` to build the files correctly, update version information, cut a tag, and deploy the files to `gh-pages` and `master`.
```
[the remark this answers is not in the document]
Comment (froboy) [reply README.md]: Yup, just old text. I've updated in https://github.com/AmericanMedicalAssociation/AMA-style-guide/pull/213/commits/719303d4c572b0c4ca0e70e1cb76772294b47c46

**Comment by froboy** [comment]
@rpama thanks for the feedback and tweaks. All good stuff. I moved the section into the docs. You can review the change and test that the link works at https://github.com/AmericanMedicalAssociation/AMA-style-guide/blob/feature/tag-releases/README.md (note, links to other docs in the repo don't work on the review tab).

**Comment by rpama** [comment]
My only remaining question is whether the "Project Branches" information better belongs in the readme vs the `docs/git-workflow.md` document since it's really most instructive for people who will be working with git on our project. But I suppose you could make a case for it being more broadly applicable information... Anyway, those are my final thoughts. LGTM & thanks for getting all this worked out, @froboy!

File: styleguide/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "ama-style-guide",
-  "version": "0.0.8",
+  "version": "1.0.1",
```
Comment (mrspete) [inline styleguide/package.json]: My only question is that I'm confused about this.  It looks like 1.0.1 was the release PRIOR to 1.0.0, which may have just been a mistake on megh's part.  we should update this to correspond with the sprint number now, correct?  Like if I'm going to cut a release on this today, I would name it v1.11.0, correct?

Comment (froboy) [reply styleguide/package.json]: Correct.

**Pull request merged**